app/views/my_modules/_recent_protocol_dropdown.html.erb

```diff
@@ -0,0 +1,22 @@
+<% display_status = protocol.description.empty? && protocol.steps.count.zero? %>
+<% if Protocol.recent_protocols(current_user, current_team, 14).count.positive? %>
```

Comment:
Is it possible to move it to controller?

Reply:
fix